packages/editor/src/client/blocks/mapLeafletBlock.tsx

```diff
@@ -0,0 +1,168 @@
+import React, {useState} from 'react'
+import {Map, TileLayer, Marker, Popup} from 'react-leaflet'
+import 'leaflet/dist/leaflet.css'
+import '../customCSS/leaflet.css'
+
+import {Icon} from 'leaflet'
+
+// stupid hack so that leaflet's images work after going through webpack
```

Comment:
Before the merge this needs to be changed

Reply:
Do you mean the comment needs to be (obviously) changed or that we have to find a more clean solution that doesn't use a hack?